infrastructure/zk/src/docker.ts

```diff
@@ -86,6 +90,23 @@ function defaultTagList(image: string, imageTagSha: string, imageTagShaTS: strin
         ? ['latest', 'latest2.0', `2.0-${imageTagSha}`, `${imageTagSha}`, `2.0-${imageTagShaTS}`, `${imageTagShaTS}`]
         : [`latest2.0`, 'latest'];
 
+    if (
+        protocolVersionTag &&
+        [
+            'proof-fri-compressor',
+            'proof-fri-gpu-compressor',
+            'prover',
+            'prover-fri',
+            'prover-fri-gateway',
+            'prover-gpu-fri',
+            'prover-v2',
```

Comment:
do we need prover-v2 and prover? seems like legacy

Reply:
Done